Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -122,7 +123,7 @@
       - id: BoxID
       - id: BoxHeadset
       - id: IDComputerCircuitboard
-      - id: WeaponDisabler
+      - id: WeaponEnergyGunMini #LPP
```

yamllint

[warning] too few spaces before comment

(comments)